src/app/settings/admin/presets/module.ts

```diff
@@ -13,12 +13,13 @@
 // limitations under the License.
 
 import {NgModule} from '@angular/core';
+import {MatChipsModule} from '@angular/material/chips';
 import {PresetListComponent} from '@app/settings/admin/presets/component';
 import {AdminSettingsPresetsRoutingModule} from '@app/settings/admin/presets/routing';
 import {SharedModule} from '@shared/module';
 
 @NgModule({
-  imports: [SharedModule, AdminSettingsPresetsRoutingModule],
+  imports: [SharedModule, AdminSettingsPresetsRoutingModule, MatChipsModule],
```

Comment:
Why is it imported here? `SharedModule` holds all shared modules, so that we don't have to add them to any other component module.

Reply:
Didn't know that we were already using this.